README.md

```diff
@@ -1,9 +1,27 @@
-# Node.js Security Working Group
+# Security Work Group
```

Comment:
Nit: `Working Group` is what all the other WGs are listed as, so should probably keep that consistent here.